keyboards/projectkb/alice/alice.c

```diff
@@ -1,27 +1,18 @@
 #include "alice.h"
 
+__attribute__ ((weak))
 void matrix_init_board(void){
```

Comment:
```suggestion
void matrix_init_kb(void) {
    matrix_init_user();
    led_init_ports();
}

void led_init_ports(void) {
```

`matrix_init_board()` doesn't seem to be an actual function in QMK, and is not called anywhere. This is the usual structure of the LED init code.

Reply:
I imagine cause the new suggestion doesn't compile - there's already a matrix_init_kb defined. Error when compiling

```
 |
 | .build/obj_projectkb_alice_madhatter/keyboards/projectkb/alice/alice.o: In function `matrix_init_kb':
 | /Users/sidcarter/qmk_firmware/keyboards/projectkb/alice/alice.c:3: multiple definition of `matrix_init_kb'
 | .build/obj_projectkb_alice_madhatter/keyboard.o:/Users/sidcarter/qmk_firmware/keyboards/cannonkeys/stm32f072/keyboard.c:86: first defined here
 | collect2: error: ld returned 1 exit status
 |
make[1]: *** [.build/projectkb_alice_madhatter.elf] Error 1
```

Reply:
Ah... the board seems to be pulling in some stuff from the cannonkeys folder, looks like for a custom underglow implementation. So this'll do for now.

Per [this comment](https://github.com/qmk/qmk_firmware/pull/7173#issuecomment-550577236) from @zvecr, the new bitbang method works on F072. I believe there is also some backlight support on ARM now.

Reply:
f072 crashes if you use the regular backlight stuff (and crashes outside of the available driver implementations). `cannonkeys/stm32f072/keyboard.c` seems to be working round this by providing backlight keycodes when `BACKLIGHT_ENABLE = no`.

Reply:
I'm guessing based on this conversation that I can resolve this? :)